cli/cmd/inject.go

```diff
@@ -119,15 +119,18 @@ func (rt resourceTransformerInject) transform(bytes []byte) ([]byte, []inject.Re
 	if len(rt.proxyOutboundCapacity) > 0 {
 		conf = conf.WithProxyOutboundCapacity(rt.proxyOutboundCapacity)
 	}
-	nonEmpty, err := conf.ParseMeta(bytes)
+
+	report, err := conf.ParseMetaAndYAML(bytes)
 	if err != nil {
 		return nil, nil, err
 	}
-	if !nonEmpty {
-		r := inject.Report{UnsupportedResource: true}
-		return bytes, []inject.Report{r}, nil
+	reports := []inject.Report{*report}
```

Comment:
It appears we can deal with a single `report` (instead of having to always wrap in an single-item array) everywhere except in the `inject_utils.go` functions. That got evident in on of my branches, so I decided to tackle that later in a separate PR, but you can do that here as well if you find it's a natural change :wink: